### PR TITLE
connectors-qa: check cdk and language tags

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Closes https://github.com/airbytehq/airbyte-internal-issues/issues/2849
We want to enforce the following new rules on the `tags` field metadata:
* A single `language:` tag should be declared
* We have a `cdk` tag to determine if the connector is a low-code one

# How
* Introduce two new checks
* Update the doc
* Make changes in https://github.com/airbytehq/airbyte/pull/35670 to make sure the new checks are not breaking all connectors CI

### CODE REVIEW

**Comment by alafanechere** [comment]
* **#35685** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/35685?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alafanechere and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by natikgadzhi — COMMENTED** [review]
Renaming connectors-qa is on your list, right?

**Comment by alafanechere** [comment]
> Renaming connectors-qa is on your list, right?

@natikgadzhi yes - is it blocking this change?

**Comment by alafanechere** [comment]
@natikgadzhi can I merge this now that https://github.com/airbytehq/airbyte/pull/35670 is mainlined?

**Review by katmarkham — APPROVED** [review]

File: airbyte-ci/connectors/connectors_qa/tests/unit_tests/test_checks/test_metadata.py [hunk 1]
```diff
@@ -155,3 +155,56 @@ def test_pass_when_java(self, mocker, tmp_path):
...
+        (code_directory / "source_test").mkdir()
+        (code_directory / "source_test"/ consts.LOW_CODE_MANIFEST_FILE_NAME).touch()
+
+        # Act
+        result = metadata.CheckConnectorCDKTag()._run(connector)
+
+        # Assert
+        assert result.status == CheckStatus.FAILED
+        assert "Expected CDK tag 'cdk:low-code'" in result.message
+        assert "but found 'cdk:python'" in result.message
```
Comment (natikgadzhi) [inline airbyte-ci/connectors/connectors_qa/tests/unit_tests/test_checks/test_metadata.py:210]:
```suggestion
        assert "but found 'cdk:python'" in result.message
        
```

**Review by natikgadzhi — APPROVED** [review]

**Pull request merged**